### PR TITLE
DOC: Clean up README

### PR DESCRIPTION
**Pull request by jarrodmillman**
You can see a rendered version here:  https://github.com/jarrodmillman/numpy/tree/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,23 @@
...
    * tools for integrating C/C++ and Fortran code
    * useful linear algebra, Fourier transform, and random number capabilities.
 
-It derives from the old Numeric code base and can be used as a replacement for Numeric. It also adds the features introduced by numarray and can be used to replace numarray.
-
-More information can be found at the website:
-
-* http://www.numpy.org
-
 After installation, tests can be run (if ``nose`` is installed) with:
```
Comment (charris) [inline README.md]: Maybe, "if  `nose` is installed, tests can be run after installation with::"

File: README.md [hunk 2]
```diff
@@ -1,34 +1,23 @@
-<div align="center">
-  <img src="http://www.numpy.org/_static/numpy_logo.png"><br>
-</div>
+# NumPy
```
Comment (charris) [inline README.md]: I miss the logo.

**Comment by jarrodmillman** [comment]
I used the svg logo and left justified it.  I am happy to change either.

File: README.md (hunk 2 above)
Comment (eric-wieser) [reply README.md]: Is there a cube-only SVG?

**Comment by jarrodmillman** [comment]
There is also a cube-only SVG:
- https://github.com/jarrodmillman/numpy/blob/readme/branding/icons/numpylogoicon.svg

**Comment by eric-wieser** [comment]
I agree - the first looks best to me too. It's a shame that the title doesn't contain selectable text, but you've set the `alt` attribute, and without any way of shifting the image margin, this seems like the best option

**Review by eric-wieser — APPROVED** [review]

**Comment by charris** [comment]
Thanks @jarrodmillman. We should probably add the license, contributor guides, etc., but this is a start.

**Pull request merged**